src/lib/util/images/index.ts

```diff
@@ -1,12 +1,17 @@
+const basePath = 'https://assets.cdn.feather-insurance.com/assets/images';
+
 const images = {
-  aid: 'https://assets.cdn.feather-insurance.com/assets/images/aid.svg',
-  bed: 'https://assets.cdn.feather-insurance.com/assets/images/bed.svg',
-  bigDog: 'https://assets.cdn.feather-insurance.com/assets/images/bigDog.svg',
-  brokenAquarium: 'https://assets.cdn.feather-insurance.com/assets/images/brokenAquarium.svg',
-  brokenGlass: 'https://assets.cdn.feather-insurance.com/assets/images/brokenGlass.svg',
-  damagedLaptop: 'https://assets.cdn.feather-insurance.com/assets/images/damagedLaptop.svg',
-  moneyIncome: 'https://assets.cdn.feather-insurance.com/assets/images/moneyIncome.svg',
-  washingMachine: 'https://assets.cdn.feather-insurance.com/assets/images/washingMachine.svg',
+  aid: `${basePath}/aid.svg`,
+  bed: `${basePath}/bed.svg`,
+  bigDog: `${basePath}/bigDog.svg`,
+  dogLiability: `${basePath}/dogLiabilityIcon.png`,
+  dogLiability2x: `${basePath}/dogLiabilityIcon2x.png`,
+  dogLiability3x: `${basePath}/dogLiabilityIcon3x.png`,
```

Comment:
Since this will grow, do you think it would be better to have some categories here?

Something like:
```
const images = {
  aid: `${basePath}/aid.svg`,
  bed: `${basePath}/bed.svg`,
  verticalThumbnails: {
    dogLiability: `${basePath}/dogLiabilityIcon.png`,
    dogLiability2x: `${basePath}/dogLiabilityIcon2x.png`,
    dogLiability3x: `${basePath}/dogLiabilityIcon3x.png`,
  }
  ...
  ...
}  
```

Reply:
Sure, will update.

Reply:
I have a work in progress that's similar here: https://github.com/getPopsure/dirty-swan/pull/234/files

Also, do you prefer having a srcSet with the 3 images or having them living as siblings as you added?
E.g.:
```
dogLiability: {
    src: `${basePath}/dogLiabilityIcon.png`,
    srcSet: {
       1x: `${basePath}/dogLiabilityIcon.png`,
       2x: `${basePath}/dogLiabilityIcon2x.png`,
       3x: `${basePath}/dogLiabilityIcon3x.png`,
    }
  }
```